src/app.html

```diff
@@ -8,5 +8,6 @@
 	</head>
 	<body data-sveltekit-preload-data="hover">
 		<div style="display: contents">%sveltekit.body%</div>
+		<script src="https://www.google.com/recaptcha/api.js" async defer></script>
```

Comment:
@billdevmaster Why did you add recaptcha here?

Do you know if it is possible to rate limit the amount of phone authentication requests purely on the backend?

This websites targets people affected by various mental conditions, and I believe that recaptcha will make it impossible for some of them to use it.

So I think we need to remvoe it and find another way to limit cost.

Reply:
the recaptcha is needed for phone verification in firebase.
and also this recaptcha is invisble, so I think it doesn't affect for user to use the website.

Reply:
ok - something to look into later then

i did see the recaptcha form popping up while testing locally

Reply:
oh. well, I made it as invisible mode for recapcha.
let me check.
Btw, could you please create a slack for flexible contact?